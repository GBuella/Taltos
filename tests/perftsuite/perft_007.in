setboard r3k2r/8/8/8/8/8/8/4K3 w kq - 0 1
echo r3k2r/8/8/8/8/8/8/4K3 w kq - 0 1
perft 1
perft 2
perft 3
perft 4
perft 5
perft 6
