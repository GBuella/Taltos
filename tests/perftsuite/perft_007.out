r3k2r/8/8/8/8/8/8/4K3 w kq - 0 1
5
130
782
22180
118882
3517770
